packages/fuelchain/src/utils/logger.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
if we're ok in getting more and more different from `typechain` code, would be nice to fix this anti-patterns pointed by eslint